### DISCUSSION TITLE
"flex-row-reverse" and "justify-content-start"

### DISCUSSION THREAD

**Discussion by Malanlk** [twbs/bootstrap#34747] [General]
I am still studying Bootstrap, I have small doubt. Someone please help me regarding this.
related screenshot attached.
![Capture1](https://user-images.githubusercontent.com/76553366/129391835-88ad500a-7522-499c-a0ae-91e00cf46370.JPG)

I just try to move whole content to left side using class "justify-content-start" but doesn't work.  This works for "justify-content-end". My question is "start" mean left side of the screen or I made any mistake here?
Thank you.

**Comment by jjulee** [comment]
I don't know what UI you want, but...
Why don't you delete the `flex-row-reverse `?
